media/css/etc/firefox/retention/thank-you-referral.scss

```diff
@@ -0,0 +1,200 @@
+@import '../../../pebbles/includes/lib';
+
+main {
+    background: #45a5b3;
+    color: #fff;
+    padding: 0 10px;
+    position: relative;
+    overflow: visible;
+
+    #confetti-canvas {
+        bottom: 0;
+        height: 100%;
+        left: 0;
+        margin: 0 auto;
+        max-width: 2200px;
+        position: absolute;
+        right: 0;
+        top: 0;
+        width: 100%;
+    }
+
+    section,
+    header {
+        margin: 0 auto;
+
+        h1 {
+            @include font-size(72px);
+
+            line-height: 1;
+
+            @media #{$mq-tablet} {
+                @include font-size(100px);
+            }
+        }
+
+        p {
+            max-width: 684px;
+        }
+    }
+
+    header {
+        padding: 70px 9px 0;
+        width: 100%;
+        max-width: 1038px;
+    }
+
+    section {
+        display: flex;
+        flex-wrap: wrap;
```

Comment:
As long as this page is only viewed by Fx users (which I believe it is, because heartbeat), this should be fine.